docs/bitcoin/quicksync.md

```diff
@@ -0,0 +1,27 @@
+# QuickSync
+
+## Overview
+QuickSync downloads a torrent for a major chunk of the blockchain from MyNode peers without verifying every block.
+It saves time by avoiding the initial blockchain download and this time benefit is significant when using an HDD but marginal with a SSD. QuickSync is turned on by default and must be disabled from the settings page while setting up to trigger fully-verified blockchain download.
+
+## Tradeoffs
+
+- **Trust vs Time**: QuickSync downloads a torrent of the Bitcoin blockchain synced by the MyNode creator. This chunk of the blockchain saves the time in downloading verifying the blockchain from scratch. This convenience has a cost. The user is trusting that MyNode is propagating a legitimate copy of the Bitcoin blockchain. To minimize trust, one should disable QuickSync and download the blockchain from scratch. In case you have already synced the blockchain using QuickSync, you can [reset the blockchain](/bitcoin/troubleshoot.html#reset-blockchain).
```

Comment:
```suggestion
- **Trust vs Time**: QuickSync downloads a torrent of the Bitcoin blockchain synced by the myNode creator. This chunk of the blockchain saves the time in downloading verifying the blockchain from scratch. This convenience has a cost. The user is trusting that myNode is propagating a legitimate copy of the Bitcoin blockchain. To minimize trust, one should disable QuickSync and download the blockchain from scratch. In case you have already synced the blockchain using QuickSync, you can [reset the blockchain](/bitcoin/troubleshoot.html#reset-blockchain).
```